.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+RELEASE_ID
```

Comment:
Hang on, if Docker ignores it then it won't make it from the build phase to the run phase, right? The goal is for it to be computed in the build phase, available to a3 when building the bundle and copying it to the persistent volume or S3, and then have the same name for that bundle file later in the run phase.

Reply:
The `.dockerignore` is for ignoring these files when running locally for testing. It ensures that the Docker image creates these files/folders instead of copying them on the `COPY` command.

Reply:
I just ssh'd into a container using this image to confirm both `RELEASE_ID` and `node_modules` are there.

Reply:
```suggestion
```
Not necessary since RELEASE_ID is only ever generated inside the docker container
(we don't need to worry about accidentally copying it from the build context into the container)